test/fixtures/keys/Makefile

```diff
@@ -628,7 +628,7 @@ rsa_public.pem: rsa_private.pem
 	openssl rsa -in rsa_private.pem -pubout -out rsa_public.pem
 
 rsa_cert.crt: rsa_private.pem
-	openssl req -new -x509 -key rsa_private.pem -config rsa_cert.cnf -out rsa_cert.crt
+	openssl req -new -x509 -days -9999 -key rsa_private.pem -config rsa_cert.cnf -out rsa_cert.crt
```

Comment:
this is a typo, I'll push a fixup, but it won't affect tests, this command is run only during regen